models/models.py

```diff
@@ -0,0 +1,26 @@
+from typing import List
+
+from pydantic import BaseModel
+
+
+class CreateMaskinportenClientBody(BaseModel):
+    name: str
+    description: str
+    scopes: List[str]
+
+
+class MaskinportenClient(BaseModel):
+    client_id: str
+    name: str
+    description: str
+    scopes: List[str]
```

Comment:
Jeg er ikke så kjent med konvensjonene i FastAPI, men det var ikke åpenbart for meg at `CreateMaskinportenClientBody` blir brukt for input mens `MaskinportenClient` blir brukt i responsen (responser har jo også en body). Skulle vi heller brukt `In` og `Out` som de gjør i [dokumentasjonen her](https://fastapi.tiangolo.com/tutorial/response-model/#add-an-output-model)?